internal/hook/item_hook_handler_test.go

```diff
@@ -710,6 +711,373 @@ func parseLabelSelectorOrDie(s string) labels.Selector {
 	return ret
 }
 
+func TestGetPodExecRestoreHookFromAnnotations(t *testing.T) {
+	testCases := []struct {
+		name             string
+		inputAnnotations map[string]string
+		expected         *velerov1api.ExecRestoreHook
+	}{
+		{
+			name:             "command is missing",
+			inputAnnotations: nil,
+			expected:         nil,
+		},
+		{
+			name: "command is empty",
+			inputAnnotations: map[string]string{
+				podRestoreHookCommandAnnotationKey: "",
+			},
+			expected: nil,
+		},
+		{
+			name: "command is a string",
+			inputAnnotations: map[string]string{
+				podRestoreHookCommandAnnotationKey: "/usr/bin/foo",
+			},
+			expected: &velerov1api.ExecRestoreHook{
+				Command: []string{"/usr/bin/foo"},
+			},
+		},
+		{
+			name: "command is a json array",
+			inputAnnotations: map[string]string{
+				podRestoreHookCommandAnnotationKey: `["a","b","c"]`,
+			},
+			expected: &velerov1api.ExecRestoreHook{
+				Command: []string{"a", "b", "c"},
+			},
+		},
+		{
+			name: "error mode is continue",
+			inputAnnotations: map[string]string{
+				podRestoreHookCommandAnnotationKey: "/usr/bin/foo",
+				podRestoreHookOnErrorAnnotationKey: "Continue",
```

Comment:
Is this case-sensitive?
I can't remember what is it in the backup hook case.

Reply:
I updated the tests to always use the const.